.github/workflows/make-protos.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate attestation libraries
+on:
+  push:
+    branches:
+      - "main"
+  schedule:
+    - cron: '0 14 * * 1'
+jobs:
+  run-make-protos:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install protoc
+        uses: arduino/setup-protoc@ab6203da1c3118e4406048171b09238ad31ad73e
```

Comment:
Do we really want the _arduino_ version of protoc?  Seems odd?

Reply:
It's not the arduino version, rather they're the creators of the setup-protoc action. It's certainly preferable to maintaining our own setup-protoc action. Presumably, they have similar codegen requirements and created the action for their needs?

Reply:
Got it.  Thanks for the explanation.